packages/connect/src/threeIdConnect.ts

```diff
@@ -13,9 +13,36 @@ import type { Subscription } from 'rxjs'
 
 import { DidProviderProxy } from './didProviderProxy'
 
-const CONNECT_IFRAME_URL = process.env.CONNECT_IFRAME_URL || 'https://app-clay.3idconnect.org'
+export type NetworkConfig = {
+  connect_iframe: string
+  manage_iframe: string
+}
+export type NetworkList = Record<string, NetworkConfig>
+
+// Base iframe urls by network
+const BASE_DEV_URL = 'https://app-dev.3idconnect.org'
+const BASE_CLAY_URL = 'https://app-clay.3idconnect.org'
+const BASE_MAIN_URL = 'https://app.3idconnect.org'
+const BASE_LOCAL_URL = `http://localhost:30001`
+const DEFAULT_MANAGE_PATH = `/management/index.html`
+
+const CONNECT_IFRAME_URL = process.env.CONNECT_IFRAME_URL || BASE_CLAY_URL
 const CONNECT_MANAGE_URL =
-  process.env.CONNECT_MANAGE_URL || 'https://app-clay.3idconnect.org/management/index.html'
+  process.env.CONNECT_MANAGE_URL || `${BASE_CLAY_URL}/management/index.html`
+
+const networkConfig = (base:string):NetworkConfig => {
+  return {
+    connect_iframe: base,
+    manage_iframe: `${base}${DEFAULT_MANAGE_PATH }`
+  }
+}
+// Configuration for each network
+const networks: NetworkList = {
+  'dev-unstable': networkConfig(BASE_DEV_URL),
+  'clay': networkConfig(BASE_CLAY_URL),
+  'main': networkConfig(BASE_MAIN_URL),
```

Comment:
Can we please use the names we use in js-ceramic here? `testnet-clay` and `mainnet`. Confusing with different naming here.